docs/user-guide/configuring.md

```diff
@@ -55,7 +55,7 @@ Setting parser options helps ESLint determine what is a parsing error. All langu
 
 By default, ESLint uses [Espree](https://github.com/eslint/espree) as its parser. You can optionally specify that a different parser should be used in your configuration file so long as the parser meets the following requirements:
 
-1. It must be an npm module installed locally.
+1. It must be a path to a module installed locally OR an object with the `parse` function.
```

Comment:
Wondering if this is redundant given the below about being "Esprima-compatible"?

Maybe we could combine these points into something like this?

> It must be an object with a `parse()` or `parseForESLint()` method, or a path to a module that exports such an object.

Don't just copy my suggestion though-- give it some thought and see if you can come up with something even better :smile: